src/util.js

```diff
@@ -23,8 +22,25 @@ export let roundRect = function(obj, precision) {
   return obj;
 };
 
-export let throttle = function(fn) {
-  let throttledFn = _.throttle(fn, cfg.throttle);
-  throttledFn.now = fn;
-  return throttledFn;
+export let maybeThrottle = function(fn) {
+  fn.maybeThrottle = true;
+  return fn;
+};
+
+export let deeply = function(fn) {
+  return function(obj, iteratee) {
+    return fn(_.mapValues(obj, function(v) {
+      return _.isPlainObject(v) ? _.deeply(fn)(v, iteratee) : v;
+    }), iteratee);
+  };
+};
+
+export let throttleDeeply = function(obj, wait) {
```

Comment:
Would be great if we can avoid throttling!

But I don't understand why it can't be done simpler? Check in `throttle` if `cfg.throttle` is 0 or `undefined` and return function as it is. Otherwise, wrap it into `throttle`. 3 extra lines and that's all or am I missing something?

It took me half an hour to understand what is going on here and another half an hour to test it and find couple bugs. Don't know how much time it took you to rename all the usages...

Reply:
The answer to your question is a question: when? when should `throttle` check? because as it is now (pre-PR) `throttle` is called whenever the module is parsed (i.e. today the `cfg.throttle` is "broken", or to be politically correct can only be changed in code, not at runtime).